rapids/rapids.sh

```diff
@@ -59,10 +54,45 @@ function execute_with_retries() {
   return 1
 }
 
+function install_dask_rapids() {
+  local base
+  base=$(conda info --base)
+  local pinned
+  pinned=${base}/conda-meta/pinned
+  local mamba_env
+  mamba_env=mamba
+  
+  # Using mamba significantly reduces the conda solve-time. Create a separate conda
+  # environment with mamba installed to manage installations.
+  conda create -y -n ${mamba_env} -c conda-forge mamba
+
+  # RAPIDS releases require fixed PyArrow versions. Unpin PyArrow to solve
+  # for new environment.
+  sed -i 's/pyarrow [\.0-9\*]*/pyarrow/g' ${pinned}
```

Comment:
Should we just delete `pyarrow` pin and append it later with `>>` instead of modifying with `sed` inline? I think it will be a little bit simpler to maintain this way.

Reply:
Ack

Reply:
One drawback to doing this is that the packages in this file are alphabetized, and appending at the end of the file would make this no longer so.

Reply:
Yeah, I think this is fine - I don't think we alphabetize them intentionally in a first place.